lua/stylua/Portfile

```diff
@@ -19,60 +19,55 @@ long_description    An opinionated code formatter for Lua 5.1, Lua 5.2 and Luau.
                     a consistent code style.
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  9a29b547d8ba3784cfc3c4db8a9da3666df79a76 \
-                    sha256  ffd1fafe3a44e3cd70f139d343149501d010c0cd2ed569243c827f917a795f77 \
-                    size    258124
+                    rmd160  58366b20168b9782dbda6ddd54b9e1d56cf89f02 \
+                    sha256  51d096232a708408f48de1535b11417ab8b49ff329b199b6476815f30dbf0c1f
```

Comment:
You are missing the `size` here.

Reply:
Doh, don't know how I missed that. Thanks, corrected.